src/test/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoServiceSpec.scala

```diff
@@ -270,9 +272,36 @@ class LeonardoServiceSpec extends TestKit(ActorSystem("leonardotest")) with Flat
     val clusterName2 = s"cluster-${UUID.randomUUID.toString}"
     val cluster2 = leo.createCluster(googleProject, clusterName2, testClusterRequest.copy(labels = Map("a" -> "b", "foo" -> "bar"))).futureValue
 
-    leo.listClusters(Map.empty).futureValue.toSet shouldBe Set(cluster1, cluster2)
+    leo.listClusters(Map.empty, true).futureValue.toSet shouldBe Set(cluster1, cluster2)
   }
 
+  it should "list all active clusters" in isolatedDbTest {
+    val googleProjectId = UUID.randomUUID
+    val googleProject = s"project-${googleProjectId.toString}"
+
+    // create a couple clusters
+    val clusterName1 = s"cluster-${UUID.randomUUID.toString}"
+    val cluster1 = leo.createCluster(googleProject, clusterName1, testClusterRequest).futureValue
+
+    val clusterName2 = s"cluster-${UUID.randomUUID.toString}"
+    val cluster2 = leo.createCluster(googleProject, clusterName2, testClusterRequest.copy(labels = Map("a" -> "b", "foo" -> "bar"))).futureValue
+
+    leo.listClusters(Map.empty, true).futureValue.toSet shouldBe Set(cluster1, cluster2)
+    leo.listClusters(Map.empty, false).futureValue.toSet shouldBe Set(cluster1, cluster2)
+
+    val clusterName3 = s"cluster-${UUID.randomUUID.toString}"
+    val cluster3 = leo.createCluster(googleProject, clusterName3, testClusterRequest.copy(labels = Map("a" -> "b", "foo" -> "bar"))).futureValue
+    leo.deleteCluster(googleProject, clusterName3)
+
+    db.inTransaction(dataAccess =>
+      dataAccess.clusterQuery.updateClusterStatus(cluster3.googleId, ClusterStatus.Deleted)
+    )
```

Comment:
I'm marking the third cluster as deleted directly in the db - not sure if there's a better way?

Reply:
Given that there isn't a LeoService method to do it, this is fine.